spec/requests/medications_spec.rb

```diff
@@ -153,51 +138,39 @@ def post_create(medication_params)
     let!(:valid_medication_params) { { name: 'Updated Medication Name' } }
 
     def put_update(medication_params)
-      put :update, params: { id: medication.id, medication: medication_params }
+      put medication_path(medication), params: { medication: medication_params }
     end
 
-    context 'when the user is logged in' do
-      include_context :logged_in_user
+    context 'when signed in' do
+      before { sign_in user }
```

Comment:
not sure if there's a functional difference between these two, but opting for consistency across the spec

Reply:
I'd go with ```before { sign_in user }``` 👍